templates/miq-template.yaml

```diff
@@ -176,6 +241,10 @@ objects:
            name: "${NAME}-region"
            persistentVolumeClaim:
              claimName: ${NAME}-region
+         -
+           name: "${NAME}-assets"
```

Comment:
Can we ship this together with the image as Nick suggested? Reducing the storage requirements when unnecessary is a huge plus.